tests/hooks.js

```diff
@@ -0,0 +1,15 @@
+const { client, data } = require(".")
+
+
+exports.mochaHooks = {
+    beforeall(done) {
+        client.login()
+        console.log(client)
```

Comment:
I have no idea how mocha works but I'm assuming that you'll want to wait for `client.login()` to complete before running `console.log` or `done()`